src/mod/python.mod/configure.ac

```diff
@@ -21,16 +21,14 @@ python_avail="false"
 
 if test "x$egg_enable_python" != "xno"; then
   if test "x$egg_with_python_config" = "x"; then
-    AC_PATH_PROGS([python_config_bin], [python3-config python-config python3.12-config python3.11-config python3.10-config python3.9-config python3.8-config])
```

Comment:
it's a good thought to make future updates easier, but two things I see here- first, I think python3-config is usually a symlink to the current version, so searching for that first should prevent the scrolling of multiple failed searches
```
checking for python3.12-config... no
checking for python3.11-config... no
checking for python3.10-config... no
checking for python3.9-config... /usr/local/bin/python3.9-config
```
and also future-proof us for python.13-config, etc. 

And second, I think by adding the extra specific versions it may just keep going down the line until it finds an older version? I think I would be in favor of not making this (and the similar following) change at all.

Reply:
i was merily suggesting, to stick with upstream, but if you see more value in adopting it more to our liking, i can do those changes.

regarding "I think python3-config is usually a symlink". thats not the case with freebsd, which was the motivation for this PR.

for freebsd it is like:
```
$ ls -la /usr/local/bin/python*
-r-xr-xr-x  1 root wheel 4680 Apr  9 03:11 /usr/local/bin/python3.9
-r-xr-xr-x  1 root wheel 3153 Apr  9 03:11 /usr/local/bin/python3.9-config
```

So, shifting python3-config detection to the top would result in:
```
checking for python3-config... no
checking for python3.12-config... no
checking for python3.11-config... no
checking for python3.10-config... no
checking for python3.9-config... /usr/local/bin/python3.9-config
```
for the test system, a vanilla `FreeBSD 14.1-RELEASE-p2`.

would you still like me to change it and shift detection of python3-config to the top of the test queue?